docs/general/polkadot-uc.md

```diff
@@ -0,0 +1,337 @@
+---
+id: polkadot-uc
+title: The Polkadot Ubiquitous Computer
```

Comment:
I personally feel this is a "gav-ism" and instead naming should be left to marketing and such before this goes live.

Reply:
Would Polkadot 2.0 be a better option? From the Marcomm Element chat, I understood Polkadot UC was preferred.

Reply:
MarComms has not yet put together a specific term, last I checked. We should not use the term "Polkadot 2.0".  I've been using the rather vague phrase "potential changes to Polkadot".

Reply:
i personally like polkadot 2.0, but yeah bill I think is taking the most sensible approach